src/containers/Teams/index.tsx

```diff
@@ -1,40 +1,53 @@
-import React, {FC, ReactNode, useEffect, useState} from 'react';
+import React, {FC, ReactNode, useCallback, useEffect, useState} from 'react';
+import {useHistory, useParams} from 'react-router-dom';
 
 import {BreadcrumbMenu, EmptyPage, FlatNavLinks, PageTitle} from 'components';
 import {getTeamMembers} from 'utils/data';
 import {NavOption} from 'types/option';
-import {TeamMember, TeamName} from 'types/teams';
+import {TeamMember, TeamName, TeamsUrlParams} from 'types/teams';
 
 import TeamMemberCard from './TeamMemberCard';
 import './Teams.scss';
 
-const teamMembers = getTeamMembers();
-
-const TEAM_NAME_FILTERS: NavOption[] = [
-  {pathname: TeamName.all, title: 'All'},
-  {pathname: TeamName.dotnetCore, title: '.NET Core'},
-  {pathname: TeamName.backEndDevelopers, title: 'Back-End Developers'},
-  {pathname: TeamName.community, title: 'Community'},
-  {pathname: TeamName.design, title: 'Design'},
-  {pathname: TeamName.devOps, title: 'DevOps'},
-  {pathname: TeamName.discordManagers, title: 'Discord Managers'},
-  {pathname: TeamName.frontEndDevelopers, title: 'Front-End Developers'},
-  {pathname: TeamName.kotlinSDK, title: 'Kotlin SDK'},
-  {pathname: TeamName.marketing, title: 'Marketing'},
-  {pathname: TeamName.newUserOperations, title: 'New User Operations'},
-  {pathname: TeamName.payments, title: 'Payments'},
-  {pathname: TeamName.projectProposals, title: 'Project Proposals'},
-  {pathname: TeamName.qa, title: 'QA'},
-  {pathname: TeamName.redditModerators, title: 'Reddit Moderators'},
-  {pathname: TeamName.research, title: 'Research'},
-  {pathname: TeamName.security, title: 'Security'},
-  {pathname: TeamName.youtube, title: 'YouTube'},
+export const TEAMS: NavOption[] = [
+  {pathname: 'all', title: TeamName.all},
```

Comment:
Is it possible to refactor the team path names as constants? Just realised that the `ProfileInfo` page might need to link to the specific team's page using the path names

Reply:
Yeah, sure. I'll put an object in constants/teams.ts and then reference those here. Cool?

Reply:
Yep sounds good